packages/core/src/components/sections/Navbar/DefaultComponents.ts

```diff
@@ -50,4 +56,5 @@ export const NavbarDefaultComponents = {
   NavbarButtons: UINavbarButtons,
   IconButton: UIIconButton,
   _experimentalButtonSignIn: ButtonSignIn,
+  _experimentalSKUMatrixSidebar: LocalSKUMatrixSidebar,
```

Comment:
We gonna fix this other case (`ButtonSignIn`) later.

```suggestion
  __experimentalSKUMatrixSidebar: LocalSKUMatrixSidebar,
```